python-package/lightgbm/engine.py

```diff
@@ -276,19 +276,26 @@ def train(params, train_set, num_boost_round=100,
     return booster
 
 
-class _CVBooster(object):
-    """Auxiliary data struct to hold all boosters of CV."""
+class CVBooster(object):
+    """CVBooster in LightGBM.
+
+    Auxiliary data struct to hold and redirect all boosters of CV.
```

Comment:
Add some words about how one can use this class. We should document that this class has the same methods (members?) as `Booster` class. All method calls are actually performed for underlying Boosters and then all returned results are returned in a list.